src/src/common.cpp

```diff
@@ -1,51 +1,81 @@
+    {0, RATE_200HZ, SX127x_BW_500_00_KHZ, SX127x_SF_6, SX127x_CR_4_7, 5000, TLM_RATIO_1_64, 2, 8},
+    {1, RATE_100HZ, SX127x_BW_500_00_KHZ, SX127x_SF_7, SX127x_CR_4_7, 10000, TLM_RATIO_1_64, 2, 8},
+    {2, RATE_50HZ, SX127x_BW_500_00_KHZ, SX127x_SF_8, SX127x_CR_4_7, 20000, TLM_RATIO_NO_TLM, 2, 10},
+    {3, RATE_25HZ, SX127x_BW_500_00_KHZ, SX127x_SF_9, SX127x_CR_4_7, 40000, TLM_RATIO_NO_TLM, 2, 10},
+    {4, RATE_4HZ, SX127x_BW_500_00_KHZ, SX127x_SF_12, SX127x_CR_4_7, 250000, TLM_RATIO_1_4, 2, 10}}; // for model recovery
+
+expresslrs_rf_pref_params_s ExpressLRS_AirRateRFperf[RATE_MAX] = {
+    {0, RATE_200HZ, -112, 4380, 3500, 2000, 2000, 5000}, // ~ 3 sync packets
+    {1, RATE_100HZ, -117, 8770, 3500, 4000, 2000, 5000},
+    {2, RATE_50HZ, -120, 17540, 3500, 6000, 2000, 5000},
+    {3, RATE_25HZ, -123, 17540, 3500, 12000, 2000, 5000},
+    {4, RATE_4HZ, -131, 239620, 30000, 60000, 0, 250}}; // this means always send sync on ch[0] as soon as we can
+#endif
+
+#if defined(Regulatory_Domain_ISM_2400)
+
+#include "SX1280Driver.h"
+extern SX1280Driver Radio;
+
+expresslrs_mod_settings_s ExpressLRS_AirRateConfig[RATE_MAX] = {
+    {0, RATE_500HZ, SX1280_LORA_BW_0800, SX1280_LORA_SF5, SX1280_LORA_CR_4_5, 2000, TLM_RATIO_1_128, 2, 12},
+    {1, RATE_250HZ, SX1280_LORA_BW_0800, SX1280_LORA_SF6, SX1280_LORA_CR_LI_4_7, 4000, TLM_RATIO_1_64, 2, 14},
+    {2, RATE_150HZ, SX1280_LORA_BW_0800, SX1280_LORA_SF7, SX1280_LORA_CR_LI_4_7, 6666, TLM_RATIO_1_32, 2, 12},
+    {3, RATE_50HZ, SX1280_LORA_BW_0800, SX1280_LORA_SF9, SX1280_LORA_CR_4_5, 20000, TLM_RATIO_NO_TLM, 2, 12}};
 
 expresslrs_rf_pref_params_s ExpressLRS_AirRateRFperf[RATE_MAX] = {
-    {RATE_200HZ, -112, 4380, 2500, 1000, 2000, 5000}, // ~ 3 sync packets
-    {RATE_100HZ, -117, 8770, 2500, 2000, 2000, 5000},
-    {RATE_50HZ, -120, 17540, 2500, 4000, 2000, 5000},
-    {RATE_25HZ, -123, 17540, 5000, 8000, 2000, 5000},
-    {RATE_4HZ, -131, 239620, 30000, 60000, 0, 250}}; // this means always send sync on ch[0] as soon as we can 
+    {0, RATE_500HZ, -112, 4380, 3500, 1000, 2000, 5000}, // ~ 3 sync packets
+    {1, RATE_250HZ, -112, 4380, 3500, 1000, 2000, 5000}, // ~ 3 sync packets
+    {2, RATE_150HZ, -117, 8770, 3500, 4000, 2000, 5000},
+    {3, RATE_50HZ, -120, 17540, 3500, 6000, 2000, 5000}}; // this means always send sync on ch[0] as soon as we can
+
+#endif
+
+expresslrs_mod_settings_s *get_elrs_airRateConfig(int8_t index);
+//const expresslrs_mod_settings_s * ExpressLRS_nextAirRate;
+expresslrs_mod_settings_s *ExpressLRS_currAirRate;
+expresslrs_mod_settings_s *ExpressLRS_prevAirRate;
 
-ICACHE_RAM_ATTR expresslrs_mod_settings_s *get_elrs_airRateConfig(expresslrs_RFrates_e rate)
+ICACHE_RAM_ATTR expresslrs_mod_settings_s *get_elrs_airRateConfig(int8_t index)
 {
     // Protect against out of bounds rate
-    if (rate < 0)
+    if (index < 0)
     {
         // Set to first entry in the array (200HZ)
```

Comment:
Comments prob need updating / remove the 200hz since we added more

Reply:
fixed